build.sbt

```diff
@@ -292,23 +303,23 @@ lazy val edu_gemini_seqexec_model = crossProject(JVMPlatform, JSPlatform)
     libraryDependencies += JavaTimeJS.value
   )
 
-lazy val edu_gemini_seqexec_model_JVM:Project = edu_gemini_seqexec_model.jvm
+lazy val seqexec_model_JVM:Project = seqexec_model.jvm
 
-lazy val edu_gemini_seqexec_model_JS:Project = edu_gemini_seqexec_model.js
+lazy val seqexec_model_JS:Project = seqexec_model.js
 
-lazy val edu_gemini_seqexec_engine = project
-  .in(file("modules/edu.gemini.seqexec.engine"))
+lazy val seqexec_engine = project
+  .in(file("modules/seqexec/engine"))
   .enablePlugins(AutomateHeaderPlugin)
   .enablePlugins(GitBranchPrompt)
-  .dependsOn(edu_gemini_seqexec_model_JVM)
+  .dependsOn(seqexec_model_JVM)
   .settings(commonSettings: _*)
   .settings(
     addCompilerPlugin(Plugins.kindProjectorPlugin),
     addCompilerPlugin(Plugins.paradisePlugin),
     libraryDependencies ++= Seq(Fs2, CatsEffect.value, Log4s) ++ Monocle.value
   )
 
-lazy val edu_gemini_epics_acm = project
+lazy val acm = project
   .in(file("modules/edu.gemini.epics.acm"))
```

Comment:
If I change this to `.in(file("modules/acm"))` sbt doesn't know what java files to compile and it fails